test/integration/find_spec.mjs

```diff
@@ -93,7 +93,10 @@ describe("find bar", () => {
           await page.waitForSelector(".xfaLayer .highlight");
           const resultElement = await page.waitForSelector("#findResultsCount");
           const resultText = await resultElement.evaluate(el => el.textContent);
-          expect(resultText).toEqual("1 of 1 match");
+          /** Unicode bidi isolation characters. */
+          const FSI = "\u2068";
+          const PDI = "\u2069";
+          expect(resultText).toEqual(`${FSI}1${PDI} of ${FSI}1${PDI} match`);
```

Comment:
Sorry, but I really don't understand these change; why is this necessary?

Reply:
Because Fluent adds these markers by default, but it can be changed when creating the bundle:
https://github.com/projectfluent/fluent.js/blob/f07305556f92bf19acfccc9621adf99dd8f434d8/fluent-bundle/src/bundle.ts#L61-L66
TBH, I don't know if we should set `useIsolating` to false or not, so it's why I kept the default.